build.gradle

```diff
@@ -76,8 +89,8 @@ dependencies {
     compile 'org.scalanlp:breeze-natives_' + scalaMajorVersion + ':0.11.2'
```

Comment:
Let's add another variable for `breezeVersion` and set it to `0.12` for the 2.0 case.